File: Groove.SP/Groove.SP.Core/Entities/IntegrationLogModel.cs
using Groove.SP.Core.Models;
using System;

namespace Groove.SP.Core.Entities
{
    public class IntegrationLogModel : Entity
    {
        public long Id { get; set; }
        public string Profile { get; set; }
        public string APIName { get; set; }
        public string APIMessage { get; set; }
        public string EDIMessageType { get; set; }
        public string EDIMessageRef { get; set; }
        public DateTime PostingDate { get; set; }
        public IntegrationStatus Status { get; set; }
        public string Remark { get; set; }
        public string Response { get; set; }
    }
}

File: Groove.SP/groove-sp-web/src/app/features/integration-log/models/integration-log.model.ts
export interface IntegrationLogModel {
    name: string;
    profile: string;
    aPIName: string;
    aPIMessage: string;
    eDIMessageType: string;
    eDIMessageRef: string;
    postingDate: string;
    status: IntegrationLogStatus;
    remark: string;
    response: string;
    responseJsonPreview: string;
}

export enum IntegrationLogStatus {
    Succeed = 1,
    Failed = 2
}
